catalog-items.html

```diff
@@ -172,75 +214,94 @@ <h2 class="explore-heading">EXPLORE COLECTIONS</h2>
+                            <span class="item-name">SKISIRS#2</span>
                         </a>
                     </article>
                 </li>
 
                 <li class="products-item">
                     <article class="product">
                         <a href="catalog-items.html" class="product-link">
-                            <figure>
-                                <img src=".//img/item5.png" alt="picture description" width="290" loading="lazy" class="product-img">
-                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
-                            </figure>
+                            <picture>
+                                <source srcset="./img/item5.webp" type="image/webp">
+                                <source srcset="./img/item5.png" type="image/png">
+                                <img src="./img/item5.png" alt="Product Description" width="290" height="290" class="product-img" loading="lazy">
+                              </picture>
+                            <span class="item-name">SKISIRS#2</span>
                         </a>
                     </article>
                 </li>
 
                 <li class="products-item">
                     <article class="product">
                         <a href="catalog-items.html" class="product-link">
-                            <figure>
-                                <img src=".//img/item6.png" alt="picture description" width="290" loading="lazy" class="product-img">
-                                <figurecaption class="item-name">SKISIRS#02</figurecaption>
-                            </figure>
+                            <picture>
+                                <source srcset="./img/item6.webp" type="image/webp">
+                                <source srcset="./img/item6.png" type="image/png">
+                                <img src="./img/item6.png" alt="Product Description" width="290" height="290" class="product-img" loading="lazy">
+                              </picture>
+                            <span class="item-name">SKISIRS#2</span>
                         </a>
-                    </article>
+                    </article> 
                 </li>
-            </ul>      
+            </ul>
 
             <ul class="pagination"> 
                 <button class="pagination-item">
-                    &laquo;
+                    <svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M30 37L16 24.0001L30 11" stroke="white" stroke-width="4" stroke-miterlimit="10" stroke-linecap="square" stroke-linejoin="round"/>
+                    </svg>     
                 </button>
 
                 <button class="pagination-item">                    
-                    &raquo;
+                    <svg transform="scale(-1, 1)" width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These arrows are the same, so you can take it from a sprite via one symbol id and add this transform via CSS.